environments/gloo-gateway/core/gloo-platform/portal/active/gloo-platform-helm.yaml

```diff
@@ -65,6 +65,9 @@ spec:
         extAuthService:
             enabled: true
             extAuth:
+                image:
+                  registry: gcr.io/gloo-mesh
```

Comment:
This is to address an issue with the dev builds, which don't publish extAuth or rateLimiter images.